shared/rakefiles/entrypoint.rake

```diff
@@ -108,6 +108,61 @@ task :display_cluster_info => [:set_vars] do
   puts
 end
 
+desc "Display gpii/universal image SHA and link to GitHub commit that triggered the build"
+task :display_universal_image_info => [:set_vars] do
+  sh "#{@exekube_cmd} sh -c ' \
+    UNIVERSAL_CI_URL=\"https://ci.gpii.net\";
+    UNIVERSAL_REPO=\"https://github.com/gpii/universal\";
+    RELEASE_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master-release\";
+    UPSTREAM_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master\";
+    LOOKUP_BUILDS=\"20\";
+
+    PREFERENCES_IMAGE_SHA=$(kubectl -n gpii get deployment preferences -o=json 2> /dev/null | jq -r \".spec.template.spec.containers[0].image\" | grep -o \"sha256:.*\");
+    if [ \"$?\" != \"0\" ]; then
+      echo
+      echo \"Unable to retrieve data from K8s cluster!\";
+      echo \"This is most likely because you are not authenticated!\";
+      echo \"Try running \\\`rake sh[\\\"rake configure_kubectl\\\"]\\\` and then try again.\";
+      echo \"You can also run \\\`rake display_cluster_state\\\` for debug info.\";
+      echo
+      exit 1;
+    fi
+
+    echo
+    echo \"Deployed gpii/universal image SHA:\";
```

Comment:
I would mention that it's the gpii/universal image in use by preferences, in case there happens to be a discrepancy between preferences and other components that use gpii/universal.

Reply:
Makes sense!